docker/legacy-auth/env.template

```diff
@@ -90,3 +90,8 @@ FIFTYONE_TEAMS_PLUGIN_URL=http://teams-plugins:5151
 # Controls whether anonymous analytics are captured for the teams
 # application. Set to false to opt-out of anonymous analytics.
 FIFTYONE_APP_ANONYMOUS_ANALYTICS_ENABLED=true
+
+# Number of worker replicas for executing delegated operations with builtin infrastructure.
+# This should not exceed the value set in the deployment's license file for max concurrent delegated operators.
+# Set to 3 by default, meaning delegated operations can be executed without an external executor system.
+FIFTYONE_DELEGATED_OPERATOR_WORKER_REPLICAS=3
```

Comment:
we set this to 0 in helm charts... shoudl the default here be 0 as well so we don't suprise folks with additional infrastructure?

Reply:
yes we (product) flip flopped on that and we didn't return here.
in helm I set replicas to 3 but have enabled=false

Reply:
seems legit! carry on!

Reply:
@swheaton @brimoor what was the decision on this one? I thought we decided 3 would be the default, but maybe I am mistaken.

Reply:
it would be 3 if someone added the extra config file - this is fine, commit and carry on!